spec.html

```diff
@@ -35704,7 +35704,7 @@ <h1>
               1. Let _InputLength_ be the number of elements in _Input_.
               1. If _f_ &lt; 0 or _f_ &gt; _InputLength_, return ~failure~.
               1. Let _g_ be min(_e_, _f_).
-              1. If there exists an integer _i_ between 0 (inclusive) and _len_ (exclusive) such that Canonicalize(_rer_, _Input_[_rs_ + _i_]) is not the same character value as Canonicalize(_rer_, _Input_[_g_ + _i_]), return ~failure~.
+              1. If there exists an integer _i_ in the range 0 &lte; _i_ &lt; _len_ such that Canonicalize(_rer_, _Input_[_rs_ + _i_]) is not the same character value as Canonicalize(_rer_, _Input_[_g_ + _i_]), return ~failure~.
```

Comment:
```suggestion
              1. If there exists an integer _i_ in the inclusive range from 0 to _len_ such that Canonicalize(_rer_, _Input_[_rs_ + _i_]) is not the same character value as Canonicalize(_rer_, _Input_[_g_ + _i_]), return ~failure~.
```

Reply:
this range is not inclusive on the right